package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -281,15 +281,15 @@ def update_dictionary_point_in_cube_start_frame(array_simulation_particle_coordi
                                                            axis=0)
                 cube['centroid_of_particles_first_frame'] = centroid_particles_in_cube
             else:  # empty cube
-                cube['centroid_of_particles_first_frame'] = 'empty'
+                cube['centroid_of_particles_first_frame'] = None
```

Comment:
apparently not covered by tests... not sure if this is incorrect or if we _must_ or _can_ cover it but I'll mention it

Reply:
O that is a bit problematic, we really must cover this. It also means we're getting 'empty' entries being fed somewhere else 🙀 Could you place a blocking review here?

Reply:
done

Reply:
aaaaahhh I was wondering why I was getting to print in that branch but not elsewhere.. we're using multiprocessing 🤦🏽 

I'll have a quick look around but it looks like:
a) Errors aren't being propagated from child workers (hence why it didn't raise the relevant "truth of an array" error),
b) Codecov isn't picking up on coverage since it's not being properly logged by the child workers... we do have the multiprocessing thing set in coveragerc, but it's obviously not working